app/models/assessment/submission.rb

```diff
@@ -190,6 +190,26 @@ def eval_answer
     end
   end
 
+  def build_empty_answers
+    return if assessment.questions.count == answers.count
+
+    questions_without_answers = assessment.questions.select do |question|
+      question.answers.where(std_course_id: std_course.id, submission_id: self.id).empty?
+    end
+
+    questions_without_answers.each do |question|
+      default_params = { std_course_id: std_course.id, submission_id: id, question_id: question.id, content: '' }
+      case question.as_question_type
```

Comment:
Sorry, I don't understand what do you mean here, as_question_type is a string, not class